packages/web/src/pages/chat-page/ChatPage.tsx

```diff
@@ -34,15 +92,31 @@ export const ChatPage = ({ match }: RouteComponentProps<{ id?: string }>) => {
       containerClassName={styles.page}
       contentClassName={styles.pageContent}
       useSearch={false}
-      header={<ChatHeader currentChatId={chatId} />}
+      header={<ChatHeader ref={headerRef} currentChatId={currentChatId} />}
     >
       <div className={styles.layout}>
         <div className={styles.chatList}>
-          <ChatList className={styles.chatList} currentChatId={chatId} />
+          <ChatList
+            className={styles.chatList}
+            currentChatId={currentChatId}
+            onChatClicked={handleChatClicked}
+          />
         </div>
-        <div className={styles.messages}>
-          <ChatMessageList className={styles.messageList} chatId={chatId} />
-          <ChatComposer className={styles.messageComposer} chatId={chatId} />
+        <div className={styles.chatArea}>
```

Comment:
Doing the scrolling stuff was interesting. I chose using this element as the scroll area to make having the composer stick to the bottom a lot easier. The side effect is I have to move it up to slide it under the header for that nice blur effect. I can do either way, but this seemed the cleanest